src/EditorFeatures/Core/Implementation/Suggestions/FixAllGetFixesService.cs

```diff
@@ -133,7 +105,7 @@ private CodeAction GetFixAllCodeAction(FixAllProvider fixAllProvider, FixAllCont
                 newSolution = PreviewChanges(
```

Comment:
`PreviewChanges` invocation will bring up another modal dialog - I am presuming this will dismiss the wait indicator dialog within which this method is executing?